### ISSUE TITLE
Allele0, Allele1, Allele2(?)

### ISSUE THREAD

**Issue by carbocation** [omeed-maghzian/mtag#54]
The Basics in the Wiki state: "a1 is considered to be the effect allele, which should also be reflected in the signs of the Z-scores"

In BOLT-LMM output, ALLELE1 is the effect allele and ALLELE0 is the other (As of 2.3.3: "*ALLELE1: first allele in bim file (usually the minor allele), used as the effect allele*"), so I want ALLELE1 to be my effect allele (a1 according to the Wiki).

But if I pass the following arguments:
```
  --a1_name ALLELE1 \
  --a2_name ALLELE0 \
```

I then get the following message in the log:
```
ALLELE0:	Allele 2, interpreted as non-ref allele for signed sumstat.
ALLELE1:	Allele 1, interpreted as ref allele for signed sumstat.
```

This is confusing in a couple ways:
1. What is Allele 2?
1. The wiki says that a1 is the effect allele, but if I pass ALLELE1 to a1, the log tells me that is the ref allele.

I don't think this ultimately matters (if everything is reversed, it's just a sign flip at the end), but it would be nice for the language to be uniform here. Thanks for considering.

**Comment by huilisabrina** [comment]
Hi @carbocation ,

I just changed the log-file so that it better matches the description in the wiki. Sorry if this appeared confusing before! Hope this is all clear now. Now the log reads:

```
ALLELE0:	a2, interpreted as non-ref allele for signed sumstat.
ALLELE1:	a1, interpreted as ref allele for signed sumstat.
```
Thanks,
Hui

**Comment by huilisabrina** [comment]
@carbocation I see no need to attach the log :) I did not make the updates for issue #53 backward compatible with this one. Just resolved it could you pull the latest af7810eaa8f3672dc320611b66e3355176d1e207 ?

Thanks! 
Hui

**Comment by carbocation** [comment]
Thanks. Looks great and everything is working now!
